### ISSUE TITLE
How to use it?

### ISSUE THREAD

**Issue by smartsoul** [wcandillon/react-native-expo-image-cache#92]
I don't know how to use it at all.

**Comment by FrankenApps** [comment]
Me, too.
I would especially be interested if it is possible to preload local images loaded with `require('path')`?
Because I would like to resolve [this](https://stackoverflow.com/questions/53130480/image-flickers-on-first-load-in-reactnative/53130550#comment93154832_53130550) problem, I have.

**Comment by nandorojo** [comment]
Yeah, the docs aren't that useful here. I assume you make the image, then next time just check if it exists in the cachemanager and otherwise just render it yourself? Either way, really surprised there isn't a super simple walk through.

**Comment by nandorojo** [comment]
Hey all, there's actually no walk-through needed. You can use it just like the vanilla Image component.

**Comment by cielor** [comment]
> Hey all, there's actually no walk-through needed. You can use it just like the vanilla Image component.

nope, it doesnt work like that, instead of using source, use uri, example <Image uri={imageSource}>

**Comment by nandorojo** [comment]
> > Hey all, there's actually no walk-through needed. You can use it just like the vanilla Image component.
> 
> nope, it doesnt work like that, instead of using source, use uri
> example: `<Image uri={imageURL}>`

Yup, you're right, my apologies. I believe it's the preview that takes in a ``{{ uri: '' }}``, but I haven't seen it in a while.
